README.rst

```diff
@@ -40,17 +43,17 @@ Azure Pipelines     `.azure-pipelines.yml`_             |AzureStatus|
 Supported Configurations
 ========================
 
-======== === ======= ===== === ======= ===== === ======= ===== === =======
-CI           Linux APT        Linux Docker        Windows         MacOS
--------- ----------------- ----------------- ----------------- -----------
-\        C++ Fortran DPC++ C++ Fortran DPC++ C++ Fortran DPC++ C++ Fortran
-======== === ======= ===== === ======= ===== === ======= ===== === =======
-GitHub   |c|   |c|    |c|  |c|   |c|    |c|  |c|   |c|    |c|  |c|   |c|
-Circle   |c|   |c|    |c|  |c|   |c|    |c|  |c|   |c|    |c|  |x|   |x|
-AppVeyor |c|   |c|    |c|  |x|   |x|    |x|  |c|   |c|    |c|  |c|   |c|
-GitLab   |c|   |c|    |c|  |x|   |x|    |x|  |c|   |c|    |c|  |x|   |x|
-Azure    |c|   |c|    |c|  |c|   |c|    |c|  |c|   |c|    |c|  |c|   |c|
-======== === ======= ===== === ======= ===== === ======= ===== === =======
+======== =========== =========== ============ =========== ===========
```

Comment:
It's up to you, but I would replace the triple check/x with a single check/x since none of the boxes are likely to be mixed.

Reply:
Thanks for the suggestion. I will make this change.

Reply:
The change is made in the last commit.